src/blockchain_db/lmdb/db_lmdb.cpp

```diff
@@ -725,6 +728,31 @@ void BlockchainLMDB::add_block(const block& blk, const size_t& block_size, const
   if (result)
     throw0(DB_ERROR(lmdb_error("Failed to add block height by hash to db transaction: ", result).c_str()));
 
+  if (blk.major_version >= 3)
```

Comment:
er... I'm confused now. It used to be 4, and I just moved it to 3 because I saw a comment saying rct was from v3. I reverted and fixed the comment now :)